tests/e2e/restart_test.go

```diff
@@ -84,7 +84,7 @@ func (s *RestartTestData) TestVolumeDetached(c *C) {
 	c.Assert(err, IsNil, Commentf(out))
 
 	// 4. Verify detached status. Volume should be detached (within the timeout)
-	status = verification.VerifyDetachedStatus(s.volumeName, s.config.DockerHosts[1], s.config.EsxHost)
+	status = verification.PollDetachedStatus(s.volumeName, s.config.DockerHosts[1], s.config.EsxHost)
```

Comment:
> Actually, the admin CLi and docker cli ways of getting the volume status is the same - from the KV. So there's no difference in getting the status from the ESX side. Just doing a docker inspect is sufficient.

Correct and logical. However, we had run into some issues in the past where there were inconsistent status results on docker host and admin cli.
CC @shuklanirdesh82